### DISCUSSION TITLE
Length of FixedUpdate

### DISCUSSION THREAD

**Discussion by etzabo** [bevyengine/bevy#8609] [Q&A] [answered]
What exactly is the length of a fixed update and how can I modify it?
```rust
.add_systems(( // List of systems running on a regular basis:
	fixed_physics,
	pre_fixed_logic,
	fixed_logic,
	post_fixed_logic,
)
	.chain()
	.in_schedule(CoreSchedule::FixedUpdate)
)
```

**Comment by mockersf** [comment, marked as answer]
You can look at the [`fixed_timestep` example](https://github.com/bevyengine/bevy/blob/latest/examples/ecs/fixed_timestep.rs) to see how to configure it.

The default duration is [1/60th of a second](https://docs.rs/bevy/latest/bevy/time/prelude/struct.FixedTime.html#structfield.period).

**Reply by etzabo** [reply]
Awesome thanks.
